PhysicsTools/PatAlgos/python/selectionLayer1/selectedPatCandidates_cff.py

```diff
@@ -35,6 +35,18 @@
 
 selectedPatCandidates = cms.Sequence(selectedPatCandidateSummary, selectedPatCandidatesTask)
 
-from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-pp_on_AA.toReplaceWith(selectedPatCandidatesTask, selectedPatCandidatesTask.copyAndExclude([selectedPatOOTPhotons]))
-pp_on_AA.toModify(selectedPatCandidateSummary.candidates, func = lambda list: list.remove(cms.InputTag("selectedPatOOTPhotons")) )
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+HI = (pp_on_AA_2018 | pp_on_PbPb_run3)
+HI.toReplaceWith(selectedPatCandidatesTask,
+                 selectedPatCandidatesTask.copyAndExclude([selectedPatOOTPhotons]))
+HI.toModify(selectedPatCandidateSummary.candidates,
+            func = lambda list: list.remove(cms.InputTag("selectedPatOOTPhotons")))
+
+from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+mAOD = (run2_miniAOD_94XFall17 | run2_miniAOD_80XLegacy)
```

Comment:
```suggestion
_mAOD = (run2_miniAOD_94XFall17 | run2_miniAOD_80XLegacy)
```
to avoid exporting this symbol

Reply:
Done.